Temperature Convertor/TeperatureConvertor.js

```diff
@@ -0,0 +1,16 @@
+// *Challenge 3
+// Challenge:
+// Write a JavaScript program to convert temperatures to and from celsius, fahrenheit.
+
+// JavaScript: Fahrenheit and Centigrade Temperatures:
+
+// Fahrenheit and centigrade are two temperature scales in use today. The Fahrenheit scale was developed by the German physicist Daniel Gabriel Fahrenheit . In the Fahrenheit scale, water freezes at 32 degrees and boils at 212 degrees.
+
+// The centigrade scale, which is also called the Celsius scale, was developed by Swedish astronomer Andres Celsius. In the centigrade scale, water freezes at 0 degrees and boils at 100 degrees. The centigrade to Fahrenheit conversion formula is:
+
+// C = (5/9) * (F - 32)
+
+function temperatureConverter(valNum) {
+  valNum = parseFloat(valNum);
+  document.getElementById("outputCelsius").innerHTML = (valNum - 32) / 1.8;
```

Comment:
When a user deletes his number, the celsius value is NaN. Can you explain why?

Reply:
This is because the function is looking for a specific number to spit out a result. When the number is deleted, we are no longer feeding it a number, so it displays Not a Number.

Reply:
Yes, the function expects a number, but when the input is deleted, the `valNum` is `undefined` which is not a number. To prevent this you can use the `or` operator ( ` || ` ) and assign it either to `valNum` or to `0`.

Reply:
`valNum = parseFloat(valNum) || 0;`